keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -440,8 +406,22 @@ class TitusClusterHandler(
   private fun Instant.iso() =
     atZone(ZoneId.systemDefault()).format(DateTimeFormatter.ISO_DATE_TIME)
 
-  private fun TitusServerGroup.exportSpec() =
-    TitusServerGroupSpec(
+  private fun TitusServerGroup.exportSpec(application: String): TitusServerGroupSpec {
```

Comment:
This adds support for omitting defaults from Titus cluster exports, which was partly where the conversation on refactoring the export code emerged from.